<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1e1e1e;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QSplitter" name="HSpliiter">
      <property name="styleSheet">
       <string notr="true">QSplitter::handle {
    /*background-color: #808080;  设置分隔线的背景色 */
}

QSplitter::handle:hover {
    background-color: #4671f1; /* 设置鼠标悬停时的背景色 */
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QStackedWidget" name="LeadTreeStack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #252526;</string>
       </property>
       <widget class="QWidget" name="lead_default"/>
      </widget>
      <widget class="QSplitter" name="VSpliiter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSplitter::handle {
    background-color: #393939; /* 设置分隔线的背景色 */
}

QSplitter::handle:hover {
    background-color: #4671f1; /* 设置鼠标悬停时的背景色 */
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="opaqueResize">
        <bool>false</bool>
       </property>
       <property name="handleWidth">
        <number>1</number>
       </property>
       <widget class="QStackedWidget" name="user_stacked">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1e1e1e;</string>
        </property>
        <widget class="QWidget" name="user_default"/>
       </widget>
       <widget class="QTabWidget" name="Tab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget {
    border: none; /* 去除QTabWidget的边框 */
}

QTabWidget::pane {
    border: none; /* 去除QTabWidget的内容区域边框 */
}


QTabBar::tab {
    min-width: 35px; /* 设置标签宽度 */
	background-color: #1e1e1e;
}

QTabBar::tab-bar {
    min-width: 35px; /* 设置标签宽度 */
}

QTabBar::tab:!selected {
	color: #808080;
}

QTabBar::tab:selected {
	color: #ffffff;
}

QTabBar::tab:hover {
	color: #ffffff;
}


</string>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="SideBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);</string>
      </property>
      <layout class="QVBoxLayout" name="toolbar_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="plugin_list" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>999999999</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="user">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 上边框宽度0像素、突出显示、颜色为透明 */
QToolButton{   
	border: 0px outset transparent;       
	background-color: #333333;	
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: #151515;	
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深灰 */
	background-color:#151515;	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/imgs/user.png</normaloff>:/imgs/user.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="setup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 上边框宽度0像素、突出显示、颜色为透明 */
QToolButton{   
	border: 0px outset transparent;       
	background-color: #333333;	
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: #151515;	
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深灰 */
	background-color:#151515;	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/imgs/setup.png</normaloff>:/imgs/setup.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 61, 57);
color: rgb(204, 204, 204);


</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTerminal">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="new_terminal"/>
    <addaction name="log"/>
    <addaction name="leadtree"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuplugins">
    <property name="title">
     <string>plugins</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTerminal"/>
   <addaction name="menuplugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="new_terminal">
   <property name="text">
    <string>terminal</string>
   </property>
  </action>
  <action name="log">
   <property name="text">
    <string>log</string>
   </property>
  </action>
  <action name="leadtree">
   <property name="text">
    <string>leadtree</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
